### ISSUE TITLE
[Feature request] Bulk change strategy for all or selected pairs

### ISSUE THREAD

**Issue by nAffie** [Sjakil/gunthy-gui#30]
I would like to have an option to bulk change the strategy for all, or selected pairs.
When adding new pairs, it's easy to set the strategy for all pairs, but for pairs already added you have to do that manually per pair, or edit the config file to perform a find/replace action.

**Comment by Utone** [comment]
Yes I suggest adding the feature to save several configs per strategy.

A check box on the pairs panel, to select pairs for batch hot swapping config, just like you have when adding pairs via the GUI (or the same screen but for active pairs) would be handy to select pairs for a debagging config or market conditions

This feature could also be a great starting point to auto adjust buy levels for selected pairs based on market conditions.

Something like this old stepgain buy level updater which increases buy levels in bearish markets and sorts coins per volume (polo only): https://www.dropbox.com/s/jvu9fem3xqx7igp/gunbot%20stepgain%20buy%20level%20updater.zip?dl=0
but there a lot on github too:
https://github.com/zcpnate/Gunbot-Polo-Updater
https://github.com/peniscoin/GunBotScripts
https://github.com/nidkil/gunbot_config_updater


At the moment I only use BB for debagging so its easily assignable, but it can be very effective as a general strategy. I would love some of my micro profit scalping BB configs in there and a few stepgain variations I use and it would be it would be awesome to be able to bulk assign to selected pairs.

**Comment by Sjakil** [comment]
Hi nAffie!

I hope the addition of groups in version 1.0 accomplishes what you are talking about here!

Please let me know
